pywbem_mock/_cimrepository.py

```diff
@@ -2217,31 +2248,30 @@ def References(self, **params):
             if inst.path.host is None:
                 inst.path.host = self.host
 
-        return self._return_assoc_tuple(rtn_insts)
+        return rtn_insts
 
     ####################################################################
     #
     #   AssociatorNames server request
     #
     ####################################################################
 
-    def AssociatorNames(self, **params):
+    def AssociatorNames(self, ObjectName, AssocClass=None, ResultClass=None,
+                        Role=None, ResultRole=None):
         # pylint: disable=invalid-name
         """
-        Implements a mock WBEM server responder for
+        Implements a  responder for
         :meth:`~pywbem.WBEMConnection.AssociatorNames`
         """
 
         namespace = self.namespace
```

Comment:
See previous comment about `namespace = self.namespace`. Only this time, the namespace is (optionally) passed in ObjectName.